sealtk/gui/Player.cpp

```diff
@@ -229,8 +232,12 @@ void Player::initializeGL()
 {
   QTE_D();
 
-  connect(this->context(), &QOpenGLContext::aboutToBeDestroyed,
-          this, [d]{ d->destroyResources(); });
+  d->destroyResourcesConnection = connect(
+    this->context(), &QOpenGLContext::aboutToBeDestroyed,
+    this, [this] {
```

Comment:
It might be better to capture `d` instead? (It's not going to change.) Then the lambda can be a one-liner:
```
    connect(this->context(), &QOpenGLContext::aboutToBeDestroyed,
            this, [d]{ d->destroyResources(); });
```